CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ### Changes
 
 * Improves `debounce` function to handle async properly (waiting for previous async call to finish before to trigger new one).
+* Allows passing of the `_id` to the `edit()` method `copyOf` argument while still allowing the entire `copyOf` object for BC
```

Comment:
Something funny about this sentence, still does not mention `copyOfId`

Reply:
Rephrased and fixed.